### PR TITLE
Add Shipping On Niben

### PR DESCRIPTION
**Pull request by charabaruk**
* Add plugin
  - To Better Cities section

* Add location
  - https://www.nexusmods.com/oblivion/mods/41134/

* Add load order
  - Load after (Open) Better Cities where Niben River is touched

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -22755,6 +22755,19 @@ plugins:
...
+      - link: 'https://www.nexusmods.com/oblivion/mods/41134/'
+        name: 'Shipping On Niben'
+    after:
+      - 'Better Cities FULL.esp'
+      - 'Better Cities BRAVIL.esp'
+      - 'Better Cities LEYAWIIN.esp'
+      - 'Better Imperial City.esp'
+      - 'Better Cities IC Waterfront.esp'
+      - 'Better Cities - IC Imperial Isle.esp'
+      - 'Better Cities - Open Better Cities.esp'
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
      - 'Better Cities FULL.esp'
      - 'Better Cities - IC Imperial Isle.esp'
      - 'Better Cities LEYAWIIN.esp'
```

These are the only three plugins in this list that actually conflict with the mod (`Better Cities - Open Better Cities.esp` seems to have been dropped in favor of an in-game configuration system, so no need to add a rule for it anymore). I've also sorted them alphabetically (i.e., ignoring characters like hyphens unless they break ties).

**Review by sibir-ine — CHANGES_REQUESTED** [review]

File: masterlist.yaml (hunk 1 above)
Comment (charabaruk) [reply masterlist.yaml]: Shipping On Niben was designed for an older version of Better Cities which still did include `Better Cities - Open Better Cities.esp` so I am loathe to remove that from afters. The rest, on the other hand, no problem.

**Comment by charabaruk** [comment]
Trimmed afters.

File: masterlist.yaml [hunk 2]
```diff
@@ -22755,6 +22755,16 @@ plugins:
...
 
+  - name: 'ShippingOnNiben.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/41134/'
+        name: 'Shipping On Niben'
+    after:
+      - 'Better Cities FULL.esp'
+      - 'Better Cities - IC Imperial Isle.esp'
+      - 'Better Cities LEYAWIIN.esp'
+      - 'Better Cities - Open Better Cities.esp'
```
Comment (sibir-ine) [inline masterlist.yaml:22766]:
```suggestion
```

I checked with v6.1.0 of BC, & the files don't actually conflict after all.

**Comment by charabaruk** [comment]
Removed OBC.

**Review by sibir-ine — APPROVED** [review]

**Pull request merged**